docs/source/solving.rst

```diff
@@ -1,2 +1,4 @@
 Solving
-=======
\ No newline at end of file
+=======
+.. automodule:: gtep.gtep_solution
+    :members: ExpansionPlanningSolution
```

Comment:
Nothing gets displayed here because there are no docstrings in the `ExpansionPlanningSolution` class